Fix boost python component name. As this is highly distribution-dependent, let
robotpkg pass the correct library name.

--- cmake/boost.cmake~	2018-05-13 15:39:08.000000000 +0200
+++ cmake/boost.cmake	2018-05-28 13:48:50.613651605 +0200
@@ -67,7 +67,7 @@
         ENDIF(NOT PYTHONLIBS_FOUND)
 
         LIST(REMOVE_AT BOOST_COMPONENTS_ ${PYTHON_IN_BOOST_COMPONENTS})
-        SET(BOOST_PYTHON_COMPONENT "python${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
+        SET(BOOST_PYTHON_COMPONENT "python")
         LIST(APPEND BOOST_COMPONENTS_ ${BOOST_PYTHON_COMPONENT})
     ELSE(Boost_VERSION VERSION_GREATER 106699)
       # Check if Python has been found
@@ -77,7 +77,7 @@
       IF(${PYTHON_VERSION_MAJOR} EQUAL 3) 
         LIST(REMOVE_AT BOOST_COMPONENTS_ ${PYTHON_IN_BOOST_COMPONENTS})
         IF(${UNIX})
-          SET(BOOST_PYTHON_COMPONENT "python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
+          SET(BOOST_PYTHON_COMPONENT "python")
         ELSE(${UNIX})
           SET(BOOST_PYTHON_COMPONENT "python${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
         ENDIF(${UNIX})
